src/integrationTest/java/uk/gov/hmcts/reform/juddata/config/CamelConfig.java

```diff
@@ -70,16 +75,43 @@ ArchiveAzureFileProcessor azureFileProcessor() {
 
     @Bean
     public DataSource dataSource() {
-        final PGSimpleDataSource ds = new PGSimpleDataSource();
-        ds.setUrl(testPostgres.getJdbcUrl());
-        ds.setUser(testPostgres.getUsername());
-        ds.setPassword(testPostgres.getPassword());
-        return ds;
+        DataSourceBuilder dataSourceBuilder = DataSourceBuilder.create();
+        dataSourceBuilder.driverClassName("org.postgresql.Driver");
+        dataSourceBuilder.url(testPostgres.getJdbcUrl());
+        dataSourceBuilder.username(testPostgres.getUsername());
+        dataSourceBuilder.password(testPostgres.getPassword());
+        return dataSourceBuilder.build();
+    }
+
+
+    @Bean("springJdbcDataSource")
+    public DataSource springJdbcDataSource() {
+        DataSourceBuilder dataSourceBuilder = DataSourceBuilder.create();
+        dataSourceBuilder.driverClassName("org.postgresql.Driver");
+        dataSourceBuilder.url(testPostgres.getJdbcUrl());
+        dataSourceBuilder.username(testPostgres.getUsername());
+        dataSourceBuilder.password(testPostgres.getPassword());
+        return dataSourceBuilder.build();
+    }
+
+    @Bean("springJdbcTemplate")
+    JdbcTemplate springJdbcTemplate() {
+        JdbcTemplate jdbcTemplate = new JdbcTemplate();
+        jdbcTemplate.setDataSource(springJdbcDataSource());
+        return jdbcTemplate;
     }
 
     @Bean(name = "txManager")
     public PlatformTransactionManager txManager() {
-        PlatformTransactionManager platformTransactionManager = new DataSourceTransactionManager(dataSource());
+        DataSourceTransactionManager platformTransactionManager = new DataSourceTransactionManager(dataSource());
+        platformTransactionManager.setDataSource(dataSource());
+        return platformTransactionManager;
+    }
+
+    @Bean(name = "springJdbcTransactionManager")
+    public PlatformTransactionManager springJdbcTransactionManager() {
+        DataSourceTransactionManager platformTransactionManager = new DataSourceTransactionManager(springJdbcDataSource());
+        platformTransactionManager.setDataSource(springJdbcDataSource());
         return platformTransactionManager;
     }
```

Comment:
can these beans be given better name? it  is not clear where the jdbc template or data source or transaction manager is used
Name should be self explanatory so that its easy to understand the code and easy to maintain